### DISCUSSION TITLE
Add drawing with drauu

### DISCUSSION THREAD

**Discussion by codingluke** [marp-team/marp#448] [Ideas] [answered]
I am actively following sli.dev's progression. Wherease I totally like the simpliness of marp (I am not a big fan of html or vue tags as content), I do miss some handy feature from slidev. One is the ability to draw on the presentations in presenter mode. As a teacher this would be a total game changer :D

They even extracted this feature to a vanilla library. https://github.com/antfu/drauu

Are there any plans on such an integration. When not, as I guess, do you see any chance to add this as a plugin?

**Comment by yhatt** [comment, marked as answer]
No. Every tools by Marp team are focusing to features _for making presentation slides rather than helping presenters_. A HTML exported from Marp CLI should be simple and minimum, so currently we have no plans to add more features.
(I have learned from [a prior Marp app](https://yhatt.github.io/marp/) that maintaining a lot of out-of-scope features from our project leads burning out 🔥)

For getting much more presentation features like drawing, we always recommend to build your presentation environment based on the library/framework that has designed for enhancing presenter, and integrate [Marp Core](https://github.com/marp-team/marp-core) with it. For example, [slides rendered by Marp Core can put into Slidev](https://poc-marp-slidev.netlify.app/).

- https://github.com/yhatt/poc-marp-slidev
- https://twitter.com/y_hatt/status/1599472080227274752

Related: #441
